app/models/service/containers.rb

```diff
@@ -24,6 +24,10 @@ def self.start_container!(wizard_state)
       end
     end
 
+    def self.pull_image(container)
+      container.compute_resource.create_image(:fromImage => container.repository_pull_url)
```

Comment:
It worked for me with a non-standalone registry (the hub). I think that as long as you send an authenticated request to the docker client saying "pull this" it's OK - or so it seems.